contrib/local-environment/keycloak/oauth2-proxy.json

```diff
@@ -0,0 +1,54 @@
+{
```

Comment:
So it looks like the scripts are meant to load this into Keycloak, does it use some sort of database? Can this not be statically loaded from a file?

Reply:
Got rid of all scripts in favor of static file.